packages/fuel-indexer-database/postgres/src/lib.rs

```diff
@@ -198,29 +205,36 @@ pub async fn type_id_list_by_name(
     #[cfg(feature = "metrics")]
     METRICS.db.postgres.type_id_list_by_name_calls.inc();
 
-    Ok(sqlx::query(&format!(r#"
-        SELECT id, schema_version, schema_name, schema_identifier, graphql_name, table_name 
-        FROM graph_registry_type_ids 
-        WHERE schema_name = '{namespace}' AND schema_version = '{version}' AND schema_identifier = '{identifier}'"#
-    ))
+    let rows = sqlx::query(
+        "SELECT 
+            id, 
+            schema_version, 
+            schema_name, 
+            table_name, 
+            graphql_name, 
+            schema_identifier
+        FROM graph_registry_type_ids
+        WHERE schema_name = $1 AND schema_version = $2 AND schema_identifier = $3",
```

Comment:
@rvmelkonian 
- I see we switched a lot of the string formatting to `.bind()`
- That's cool if these values are more complex types (e.g. a byte array) but for a simple string, is there any reason we're using `.bind()` over the string formatting?

Reply:
the main reason is to prevent SQL injection, `format!` doesn't guarantee that kind of safety, also .bind() allows sqlx to do type checking on the values being passed to the query.